packages/expo-cli/src/commands/build/index.js

```diff
@@ -26,30 +26,27 @@ export default (program: any) => {
       'Revoke credentials on developer.apple.com, select appropriate using --clear-* options.'
     )
     .option(
-      '--apple-id <login>',
-      'Apple ID username (please also set the Apple ID password as EXPO_APPLE_PASSWORD environment variable).'
+      '--apple-id [id]',
+      'Apple ID username. Set your Apple ID password as EXPO_APPLE_PASSWORD env variable.'
     )
     .option(
-      '-t --type <build>',
-      'Type of build: [archive|simulator].',
+      '-t --type [type]',
+      'Select the type of build: [archive or simulator]',
       /^(archive|simulator)$/i,
       'archive'
     )
-    .option('--release-channel <channel-name>', 'Pull from specified release channel.', 'default')
-    .option('--no-publish', 'Disable automatic publishing before building.')
+    .option('--release-channel [channel]', 'Pull bundle from specified release channel.', 'default')
```

Comment:
I believe this affects both the release channel the app gets OTA updates from *and* the release channel the update is published to, during build (unless `--no-publish` specified)